tensorflow/python/kernel_tests/check_ops_test.py

```diff
@@ -852,6 +885,12 @@ def test_empty_tensor_doesnt_raise(self):
       out = array_ops.identity(empty)
     self.evaluate(out)
 
+  def test_static_check_in_graph_mode(self):
+    with context.graph_mode():
```

Comment:
Don't use context.graph_mode(), use ops.Graph().as_default() (which implies graph mode and is more specific)

Reply:
Please address this comment (replace context.graph_mode() with ops.Graph().as_default())

Reply:
Sorry, didn't notice this comment the last time I viewed this PR. I've pushed a commit that replaces `context.graph_mode()` with `ops.Graph().as_default()`